Project-1/README.md

```diff
@@ -13,63 +13,226 @@ Note: The standard interpretation of the logical symbols - "∨", "∧", "→",
   (b) ∃x∀y∃z and ∀x∃y∀z 
+  The first claims that for something such that it is x, everything such that it is Y has a certain relationship with X and that there is something such that it is Z has a certain relationship with all Ys and some X. The second claims that for all things such that they are X, there is something such that it is Y that has a certain relationship with it and everything such that it is Z has a certain relationship with some Y and everything such that it is X.
   (c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
+  The first claims that for all things such that they are x, there is something such that is is Y in a certain relationship with it and everything that is such that it is z is in a certain relationship with all xs and some y and there is some w which is in a certain relationship with all zs and some y and all xs. In the second, it claims that there is something such that it is x where there all ys are in a certain relationship with it, and there is some z which has a certain relationship with some x and all ys and all things such that they are w are in a certain relationship with some z and all ys and some x.
 ```
 	
 7. Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
 ```
 ∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+X1>Y1, X2>Z1<Y2, X2>Y2 Arrows indicate relationship R and its directionality
 ∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+X1>Y1>Z1, X2>Z2
 ```
 	
 8. Using an online tableau proof generator - such as the one found here `https://www.umsu.de/trees/` - provide tree proofs of the following entailments, which are known as the De Morgan's laws:
   ```
   (a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+1.¬∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+2.¬∀y(¬(Pa ∧ Qa) → (¬Pa ∨ ¬Qa))(1)
+3.¬(¬(Pa ∧ Qa) → (¬Pa ∨ ¬Qa))(2)
+4.¬(Pa ∧ Qa)(3)
+5.¬(¬Pa ∨ ¬Qa)(3)
+6.¬¬Pa(5)
+7.¬¬Qa(5)
+8.Qa(7)
+9.Pa(6)
+10.¬Pa(4)	11.¬Qa(4)
+x		x
+
   (b) ∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
+  1.¬∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
+  2.¬∀y(¬(Pa ∨ Qa) → (¬Pa ∧ ¬Qa))(1)
+  3.¬(¬(Pa ∨ Qa) → (¬Pa ∧ ¬Qa))(2)
+  4.¬(Pa ∨ Qa)(3)
+  5.¬(¬Pa ∧ ¬Qa)(3)
+  6.¬Pa(4)
+  7.¬Qa(4)
+  8.¬¬Pa(5)	9.¬¬Qa(5)
+  10.Pa(8)	11.Qa(9)
+x		x
+
   (c) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+  1.¬∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+  2.¬∀y((¬Pa ∨ ¬Qa) → ¬(Pa ∧ Qa))(1)
+  3.¬((¬Pa ∨ ¬Qa) → ¬(Pa ∧ Qa))(2)
+  4.¬Pa ∨ ¬Qa(3)
+  5.¬¬(Pa ∧ Qa)(3)
+  6.Pa ∧ Qa(5)
+  7.Pa(6)
+  8.Qa(6)
+  9.¬Pa(4)	10.¬Qa(4)
+x		x
+
   (d) ∀x∀y((¬Px ∧ ¬Qx) → ¬(Px ∨ Qx))
+  1.¬∀x∀y((¬Px ∧ ¬Qx) → ¬(Px ∨ Qx))
+  2.¬∀y((¬Pa ∧ ¬Qa) → ¬(Pa ∨ Qa))(1)
+  3.¬((¬Pa ∧ ¬Qa) → ¬(Pa ∨ Qa))(2)
+  4.¬Pa ∧ ¬Qa(3)
+  5.¬¬(Pa ∨ Qa)(3)
+  6.Pa ∨ Qa(5)
+  7.¬Pa(4)
+  8.¬Qa(4)
+  9.Pa(6)	10.Qa(6)
+x		x
+
+
+
 ```
 	
 9. Using a natural deduction proof generator - such as the one found here `https://proofs.openlogicproject.org/` - provide natural deduction proofs for each of De Morgan's laws. 
+(a)
+1. ¬(¬Pa ∨ ¬Qa)	assumption
```

Comment:
This proof is incomplete and incorrect. You swap syntax for negation on the second line, it is unclear what "~v" means, and it is not obvious to me that you have proven - or even tried to prove - a conditional, which is what was required. Moreover, there is no reference in your proof of any universal derivation method. 

I am similarly concerned about the remaining proofs. I think this warrants you, me, and your reviewers having a discussion.

Reply:
@johnbeve As a reviewer, I noticed what you pointed out here. I attach my previous review here:

"Q9. (a) 3.   (Pa ∧ Qa)	1, ~v
You can not say this because this is actually to apply DeMorgan's Law (b), begging the question. 

Similarly, the third step of the proof of (b) is also begging the question. 

Also, in your proofs of (c) and (d), when you use "contradiction", for example, in the sixth step of Proof (c), I suspect DeMorgan's law is implicitly used. "

Here is Josh's reply to my review:

"For Q9, I take your point about using DeMorgan's laws there, but I am keeping my proofs there as is for two reasons (one of which I mentioned when we talked about this on Thursday). First, as I said Thursday, I think it is preferable to keep the proofs simpler in this case. If I were to change the proof with this in mind, it would only add a few subproofs from conjunctions, disjunctions, and some extra double negations and would only make the proofs longer through that when they could be proved much quicker using the negated disjunction and conjunction rules. Second, I will also note that there is nothing in the instructions that says that we can't use the laws in the proof. Given that, I prefer the simplicity and speed of the proof in question over the less potentially-question-begging proofs.

I will also note that, while I am taking you at your word that the negated conjunction and disjunction rules are De Morgan's laws, I am not sure that I am using the rules depending on each of the relevant laws when I disprove the individual ones. For instance, you note that I appeal to (b) when I prove (a), but that is not begging the question. That is just using a different law to prove the other one. They may be intended to be used as a whole system or something to that effect, but given the lack of instruction excluding this, the different laws being used to prove each other, and the benefits of simplicity and speed over a more laborious proof I have elected to keep the proofs as is."

Reply:
To say some more about my approach to these questions, I have used what I took to be the negated conjunction and disjunction rules. These are sometimes abbreviated as Dem if my memory serves correctly, so that might be part of the issue here as I noted the rules with their more descriptive, colloquial names. I have used them before in previous proofs for my graduate logic classes and I took them to be fine as moves in a given proof like this, albeit with the concern about circularity that Karl raised.

My approach was to use indirect proofs for each question. I began by looking to the conditional being claimed as universal and assuming the one scenario where that conditional would be false, namely that the first part is true while the second part is false. With that in mind, my initial two assumptions for each match that approach. From there, I aimed to use the negated conjuction and disjunction rules to get a contradiction, showing that assuming the conditional in question to be false is contradictory. This would then prove its universality, as there is no scenario where it would be false (essentially the negation of an existential quantifier version of the whole formula). This should work as a proof for what is claimed in the formulae unless there was some sort of misunderstanding regarding what you wanted us to do here.

With that in mind, I will also somewhat repeat what I said to Karl regarding the issue of circularity. I understand if you want me to do out the whole proof without using these laws, but for this my thought process was to focus on a simpler, more efficient set of proofs over ones which involved a number of subproofs involving bringing out the atomic parts of conjunctions and disjunctions. Those proofs would work, of course, but they would be much longer when using these rules which I took to be fairly common in Natural Deductions would shorten the proofs significantly without going against the instructions given for this question. Again, I understand if you want these proofs without those rules used by that was my thought process for why I submitted them with those rules used.